OutOfSchool/OutOfSchool.BusinessLogic/Models/Workshops/IncompletedWorkshops/WorkshopWithContactsDto.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.ComponentModel.DataAnnotations;
+using Microsoft.AspNetCore.Mvc;
+using OutOfSchool.BusinessLogic.Util.JsonTools;
+
+namespace OutOfSchool.BusinessLogic.Models.Workshops.IncompletedWorkshops;
+public class WorkshopWithContactsDto : WorkshopWithDescriptionDto
```

Comment:
(Code style) Please add a blank line between the namespace and the class. And in other places too (see WorkshopWithDescriptionDto, WorkshopWithTeachersDto etc.)

Reply:
Ok, I've done it